.circleci/config.yml

```diff
@@ -32,7 +35,22 @@ commands:
           key: v2-covid19-{{ .Branch }}-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
           paths:
             - ~/.npm
-
+  inject_env_var_dev_ca2:
```

Comment:
Why inject this from ci. Configs are not a ci problem. This should be managed from code only. What is the problem you are trying to fix?

Reply:
I still do not understand the purpose of this from a CI point of view.

Reply:
so, these are being use  [here](https://github.com/dialoguemd/covid-19/blob/26af4c2356caf44a02c368d8b5cc8c86afc0f14e/src/pages/rasa.tsx#L13) during the build time, if we do not export these env vars in dev the default would point the covidlflow urls to the production ones.

Reply:
Ok but this is not a CI issue, it is a build process issue. I would personally craft something so that all we do is in package.json

Reply:
package.json already has build step for dev https://github.com/dialoguemd/covid-19/blob/26af4c2356caf44a02c368d8b5cc8c86afc0f14e/package.json#L53 the problem is can we overwrite the orab's build step which is using `npm build`

Reply:
ok, just fixed it buy overwriting the build script.

Reply:
Yeah we could have injected a ENV env var too and use it in the build script. I'm ok with your solution though.

Reply:
> why is env.production an empty file 🤔

I added chmod +x permission since the CI was failing.